core/src/main/resources/hudson/node_monitors/LabelsMonitor/column.jelly

```diff
@@ -0,0 +1,39 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:s="/lib/form">
+  <style type="text/css">
+  .all_labels {
+    display: none;
+  }
+  .labels:hover .all_labels {
+    display: block;
+  }
+  </style>
+  <td align="right">
+    <div class="labels">
+      <nobr>
+        <j:set var="count" value="0"/>
+        <j:forEach var="entry" items="${data}">
+          <j:if test="${count &lt; from.maxLabelCount || from.maxLabelCount &lt; 0}">
+              <a href="${rootURL}/label/${entry.name}" tooltip="${h.escape(entry.description)}">${entry.name}</a>&amp;#32;
```

Comment:
Use `<a class=${entry.className} model-link inside" `... to fix rendering between labels